pkg/exchange/okex/convert.go

```diff
@@ -277,3 +280,34 @@ func toLocalInterval(src string) string {
 		return strings.ToUpper(w)
 	})
 }
+
+func hashStringID(s string) uint64 {
+	h := fnv.New64a()
+	h.Write([]byte(s))
+	return h.Sum64()
+}
+
+func toGlobalOrder(okexOrder *okexapi.OrderDetails, isMargin bool) (*types.Order, error) {
+	timeInForce := types.TimeInForceFOK
+	if okexOrder.OrderType == okexapi.OrderTypeIOC {
+		timeInForce = types.TimeInForceIOC
+	}
+	return &types.Order{
+		SubmitOrder: types.SubmitOrder{
+			ClientOrderID: okexOrder.ClientOrderID,
+			Symbol:        okexOrder.InstrumentID,
+			Side:          types.SideType(okexOrder.Side),
+			Type:          toGlobalOrderType(okexOrder.OrderType),
+			Quantity:      okexOrder.Quantity,
+			Price:         okexOrder.Price,
+			TimeInForce:   types.TimeInForce(timeInForce),
+		},
+		Exchange:         types.ExchangeOKEx,
+		OrderID:          hashStringID(okexOrder.OrderID),
```

Comment:
how do we convert the order ID in other cases?

Reply:
I used kucoin exchange as reference, <- wrong, 
 Right now I use toGlobalOrders() as reference.